docs/developer/eventing/sources/creating-event-sources/writing-event-source-easy-way/README.md

```diff
@@ -143,7 +143,7 @@ image, and push it to a container registry that your cluster can access.
       template:
         spec:
           containers:
-            - image: docker.io/myregistry/node-heartbeat-source:v1
+            - image: gcr.io/knative-nightly/knative.dev/eventing/cmd/heartbeats:latest
```

Comment:
Please revert this, the page actually explains how to write a source from scratch. It makes you build an heartbeat source image from scratch. Hence the original image repo is valid